src/lava/magma/runtime/message_infrastructure/message_infrastructure/csrc/channel/dds/fast_dds.cc

```diff
@@ -0,0 +1,363 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+// See: https://spdx.org/licenses/
+
+
+#include <message_infrastructure/csrc/channel/dds/fast_dds.h>
+#include <message_infrastructure/csrc/core/message_infrastructure_logging.h>
+
+#include <fastdds/rtps/transport/shared_mem/SharedMemTransportDescriptor.h>
+#include <fastdds/rtps/transport/TCPv4TransportDescriptor.h>
+#include <fastdds/rtps/transport/TCPv6TransportDescriptor.h>
+#include <fastdds/rtps/transport/UDPv4TransportDescriptor.h>
+#include <fastdds/rtps/transport/UDPv6TransportDescriptor.h>
+#include <fastrtps/Domain.h>
+
+#include <vector>
+#include <fastdds/dds/domain/DomainParticipantFactory.hpp>
```

Comment:
include order

Reply:
From lint "Should be: fast_dds.h, c system, c++ system"  -> *.h <vector> *.hpp
difficult to be: <fastdds*> <vector>
add NOLINT?